proxy/default.conf

```diff
@@ -0,0 +1,17 @@
+server {
+	listen 80;
+	listen [::]:80;
+	server_name localhost;
+
+	location / {
+		proxy_pass http://ui:3000;
+	}
+
+	location /api {
+		proxy_pass http://lexbox-api:5158;
+	}
+
+	location /v1/traces {
```

Comment:
here you have the location for traces as `/v1/traces` however in ingress it's just `/traces` shouldn't they be the same?

Reply:
yeah, I was trying to troubleshoot that as well, I thought it was supposed to be `/traces` but I was actually seeing `/v1/traces`... more work to do there to get things right.